packages/ui/components/editor/plugins/ToolbarPlugin.tsx

```diff
@@ -376,7 +376,12 @@ export default function ToolbarPlugin(props: TextEditorProps) {
         editor.registerUpdateListener(({ editorState, prevEditorState }) => {
           editorState.read(() => {
             const textInHtml = $generateHtmlFromNodes(editor).replace(/&lt;/g, "<").replace(/&gt;/g, ">");
-            props.setText(textInHtml);
+            props.setText(
```

Comment:
Enter results in <p class="editor-paragraph"><br></p> which is the paragraph margin and a new line. We only want the margin.

Multiple enter in a row don't work as expected in the email, but for this case shift-enter can be used. Wasn't able to figure out a fix for this yet